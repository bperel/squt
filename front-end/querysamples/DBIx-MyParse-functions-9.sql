SELECT EXTRACT(YEAR_MONTH FROM '1999-07-02 01:02:03');
