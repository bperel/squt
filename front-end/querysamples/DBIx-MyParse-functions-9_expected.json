{
   "Warning" : {
      "No alias" : {
         "extract" : ["function", "SELECT"]
      }
   },
   "Functions" : {
      "0" : {
         "to" : "OUTPUT",
         "Constants" : {
            "1999-07-02 01:02:03" : "1999-07-02 01:02:03",
            "INTERVAL_YEAR_MONTH" : "INTERVAL_YEAR_MONTH"
         },
         "name" : "extract",
         "alias" : "0"
      }
   }
}
